src/main/java/fitr/command/TipManager.java

```diff
@@ -0,0 +1,18 @@
+package fitr.command;
```

Comment:
Since the class is not a command anymore, maybe can consider putting it in another package? Maybe a `tip` package?